drivers/volume/kdmp/kdmp.go

```diff
@@ -571,7 +572,7 @@ func (k *kdmp) StartRestore(
 		if err != nil {
 			return nil, err
 		}
-
+		delete(pvc.Annotations, storageNodeAnnotation)
```

Comment:
Can we move it to getRestorePVCs where other annotations are deleted?

Reply:
Fixed